src/Symfony/Component/Clock/Resources/now.php

```diff
@@ -17,9 +17,19 @@
      *
      * Note that you should prefer injecting a ClockInterface or using
```

Comment:
https://github.com/symfony/symfony/blob/92ff2b20d4360b529afa53f179bcf00e0af7d678/src/Symfony/Component/Clock/Resources/now.php#L16 is not true anymore